cp22/hpi-fun2/hexhex/problem.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "",
```

Comment:
A description is mandatory.

Reply:
Ideally you also mention here that the uploading jury member should set the memory limit of 10 MiB afterwards.

Reply:
So the description is only visible for the jury?